app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/EligibilityController.scala

```diff
@@ -79,16 +79,12 @@ class EligibilityController @Inject() (
   }
 
   def getDoYouClaim: Action[AnyContent] = notEnrolled.async { implicit request =>
-    logger.info(
-      "EligibilityController.getDoYouClaim"
-    )
+    logger.info("EligibilityController.getDoYouClaim")
```

Comment:
Just makes find usages a bit clearer else have to navigate/click to it to see the logging

![image](https://github.com/hmrc/eu-subsidy-compliance-frontend/assets/3098723/7839338f-acd2-4d18-a771-e71b107bac36)